source/overview/security.rst

```diff
@@ -2,74 +2,133 @@
+
+Users can mandate `multi-factor authentication <https://docs.mattermost.com/deployment/auth.html>`_ on Mattermost Enterprise Editions E10 and E20.
 
 Access Control Policy
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   - Limit communications to specific users, private channels, or team-wide public channels
-   - Increase system security `by restricting email-based account creation to email addresses from a list of specific domains, <https://docs.mattermost.com/administration/config-settings.html#restrict-account-creation-to-specified-email-domains>`_ e.g. "corp.mattermost.com", "mattermost.org", etc."
-   - Choose whether to restrict or enable `cross-origin requests. <https://docs.mattermost.com/administration/config-settings.html#enable-cross-origin-requests-from>`_
-   - If sharing of public links for account creation or sharing of files and images are enabled, links can be invalidated via the System Console by `regenerating salts <https://docs.mattermost.com/administration/config-settings.html#public-link-salt>`_.
-   - Optionally restrict `creation, renaming, archiving of channels, private channels and integrations to team admins, system admins or end users <https://docs.mattermost.com/administration/config-settings.html#policy-enterprise>`_ (E10)
-   - Optionally restrict `sending team invites to team admins, system admins or end users <https://docs.mattermost.com/administration/config-settings.html#policy-enterprise>`_ (E10)
-   - Optionally add `advanced passwords requirements <https://docs.mattermost.com/administration/config-settings.html#password-requirements-enterprise>`_ with minimum numbers of symbols, numbers, and lower and uppercase letters (E10).
+All of Mattermost's editions can limit communications to specific users, 
+private channels, 
+or team-wide public channels.
+System security can be increased `by restricting email-based account creation to email addresses from a list of specific domains, <https://docs.mattermost.com/administration/config-settings.html#restrict-account-creation-to-specified-email-domains>`_ , 
+for example, *corp.mattermost.com*.
+
+Users can choose whether to restrict or enable `cross-origin requests. <https://docs.mattermost.com/administration/config-settings.html#enable-cross-origin-requests-from>`_
+
+Links can be invalidated through the System Console by `regenerating salts <https://docs.mattermost.com/administration/config-settings.html#public-link-salt>`_ if sharing of public links for account creation or sharing of files and images is enabled.
+
+Administrators can:
+- restrict `creation, 
+renaming, 
+archiving of channels, 
+private channels and integrations to team admins, 
```

Comment:
0/5 - we could probably remove private channels to simplify a bit since it's covered by "channels"

Reply:
done